app/src/main/java/ju/video/player/view/commons/ComboBox.java

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright (c) 2022 Jala University.
+ *
+ * This software is the confidential and proprieraty information of Jalasoft
+ * ("Confidential Information"). You shall not disclose such Confidential
+ * Information and shall use it only in accordance with the terms of the
+ * Licence agreement you entered into with Jalasoft
+ */
+package ju.video.player.view.commons;
+
+import javax.swing.JComboBox;
+
+import java.awt.event.ItemEvent;
+import java.awt.event.ItemListener;
+
+/**
+ * It is for create a comboBox with its respective features.
+ *
+ * @author Adriana Olivera
+ * @version 1.0
+ */
+
+public class ComboBox extends JComboBox<String> implements ItemListener {
+    
+    public ComboBox(String[] itemsList) {
+        addItems(itemsList);
+        setBounds(10,10,80,20);
```

Comment:
Perhaps you might consider saving those values in final variables if they are not going to change.
You are missing a space between the comma and the number.